src/interfaces/IButteredBread.sol

```diff
@@ -11,6 +11,8 @@ interface IButteredBread {
     error InvalidValue();
     /// @notice Occurs when attempting a deposit with a non-sanctioned LP
     error NotAllowListed();
+    /// @notice Occurs when attempting to delegate ButteredBread that is determined by BREAD
```

Comment:
I was sort of confused when reading this comment. Would suggest making it more clear. Suggestion:
`Occurs when attempting to delegate ButteredBread. Delegations are set via the $BREAD contract.`